packages/hello-gsm-calculate/src/components/ScoreResultModal/index.tsx

```diff
@@ -30,8 +30,8 @@ const ScoreResultModal: React.FC<ResultType> = ({ result }) => {
         </S.SubDesc>
         <S.ConfirmSection>
           <S.Confirm onClick={() => setShowScoreResult()}>수정</S.Confirm>
-          <Link href="/mypage" passHref>
-            <S.Confirm>확인</S.Confirm>
+          <Link href="/" passHref>
+            <S.Confirm onClick={() => setShowScoreResult()}>확인</S.Confirm>
```

Comment:
```suggestion
            <S.Confirm onClick={setShowScoreResult}>확인</S.Confirm>
```

arrow function을 제거하는 것도 좋아 보입니다!